src/components/plugins/export/util.js

```diff
@@ -0,0 +1,41 @@
+const exportData = (exportFileName, data, columnModel) => {
+  let CSV = "";
+  let header = "";
+
+  columnModel.forEach((element) => {
+    if (element.export !== false) {
+      header += `${element.name},`;
+    }
+  });
+
+  header = header.slice(0, -1);
+  CSV += `${header}\r\n`;
+
+  for (let i = 0; i < data.length; i += 1) {
+    let rowData = "";
+    for (let j = 0; j < columnModel.length; j += 1) {
+      if (columnModel[j].export !== false) {
+        const value = columnModel[j].getValue
+          ? columnModel[j].getValue(data[i][columnModel[j].dataIndex])
+          : data[i][columnModel[j].dataIndex];
+        rowData += `${value},`;
```

Comment:
Need to handle special characters and prevent CSV injection. I suggest we use a library rather than do this ourselves. Might be easiest to ask the server to return the CSV content.

Reply:
added some comments, this export logic was there with the exportControl component, I moved it out to a ultil